manim/utils/color.py

```diff
@@ -16,21 +16,244 @@
     "random_bright_color",
     "random_color",
     "get_shaded_rgb",
+    "DARK_BLUE",
+    "DARK_BROWN",
+    "LIGHT_BROWN",
+    "BLUE_E",
+    "BLUE_D",
+    "BLUE_C",
+    "BLUE",
+    "BLUE_B",
+    "BLUE_A",
+    "TEAL_E",
+    "TEAL_D",
+    "TEAL_C",
+    "TEAL",
+    "TEAL_B",
+    "TEAL_A",
+    "GREEN_E",
+    "GREEN_D",
+    "GREEN_C",
+    "GREEN",
+    "GREEN_B",
+    "GREEN_A",
+    "YELLOW_E",
+    "YELLOW_D",
+    "YELLOW_C",
+    "YELLOW",
+    "YELLOW_B",
+    "YELLOW_A",
+    "GOLD_E",
+    "GOLD_D",
+    "GOLD_C",
+    "GOLD",
+    "GOLD_B",
+    "GOLD_A",
+    "RED_E",
+    "RED_D",
+    "RED_C",
+    "RED",
+    "RED_B",
+    "RED_A",
+    "MAROON_E",
+    "MAROON_D",
+    "MAROON_C",
+    "MAROON",
+    "MAROON_B",
+    "MAROON_A",
+    "PURPLE_E",
+    "PURPLE_D",
+    "PURPLE_C",
+    "PURPLE",
+    "PURPLE_B",
+    "PURPLE_A",
+    "WHITE",
+    "BLACK",
+    "LIGHT_GRAY",
+    "LIGHT_GREY",
+    "GRAY",
+    "GREY",
+    "DARK_GREY",
+    "DARK_GRAY",
+    "DARKER_GREY",
+    "DARKER_GRAY",
+    "GREY_BROWN",
+    "PINK",
+    "LIGHT_PINK",
+    "GREEN_SCREEN",
+    "ORANGE",
 ]
 
-
+from enum import Enum
 import random
 
 from colour import Color
 import numpy as np
 
-from ..constants import COLOR_MAP
-from ..constants import WHITE
 from ..utils.bezier import interpolate
 from ..utils.simple_functions import clip_in_place
 from ..utils.space_ops import normalize
 
 
+class Colors(Enum):
+    """A list of pre-defined colors.
+    
+    Examples
+    --------
+    The preferred way of using these colors is
+    
+    .. code-block:: python
+       import manim.utils.color as C
+       C.WHITE   # -> '#FFFFFF'
```

Comment:
I am a little late to this party, but whatever, just as a general FYI: if this would have been written as

```
.. code-block:: python
    
    >>> import manim.utils.color as C
    >>> C.WHITE
    '#FFFFFF'
```

then it would be a doctest, and these two lines would have been tested with every run of the pipeline.

Reply:
@kolibril13 let's do this :)

Reply:
https://github.com/ManimCommunity/manim/pull/496  ;)
Btw. does this mean, that the test can be removed?
All in all, I think we will have some redundancy when we have tests in the test folder and tests in the docs.

Reply:
My rule of thumb for tests is: if it is easy to setup and maybe even illustrates something useful for users, then it can be a doctest.

Stuff that needs more setup is certainly better suited in dedicated test files. :-)